docs/content/latest/explore/indexes-constraints/overview.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Overview
+linkTitle: Overview
+description: Overview of Indexes in YSQL and YCQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: overview
+    parent: explore-indexes-constraints
+    weight: 200
+aliases:
+   - /latest/explore/ysql-language-features/indexes-1/
+   - /latest/explore/indexes-constraints/indexes-1/
+isTocNested: true
+showAsideToc: true
+
+---
+
+The use of indexes can enhance database performance by enabling the database server to find rows faster. You can create, drop, and list indexes, as well as use indexes on expressions.
+
+## Create indexes
+
+You can create indexes in YSQL and YCQL using the `CREATE INDEX` statement that has the following syntax:
+
+```sql
+CREATE INDEX index_name ON table_name(column_list);
+```
+
+*column_list* represents a column or a comma-separated list of several columns to be stored in the index. An index created for more than one column is called a composite index.
+
+You can also create a functional index in YSQL, in which case you would replace any element of *column_list* with an expression. For more information, see [Expression Indexes](../../../explore/indexes-constraints/expression-index-ysql/).
+
+YSQL currently supports index access methods `lsm` (log-structured merge-tree) and `ybgin`. These indexes are based on YugabyteDB's DocDB storage and are similar in functionality to PostgreSQL's `btree` and `gin` indexes, respectively. The index access method can be specified with `USING <access_method_name>` after *table_name*. By default, `lsm` is chosen. For more information on `ybgin`, see [Generalized inverted index](../../../explore/indexes-constraints/gin/).
+
+You can apply sort order on the indexed columns as `ASC` (default), `DESC`, as well as `HASH`. For examples, see [HASH and ASC examples in YSQL](../../../api/ysql/the-sql-language/statements/ddl_create_index/#unique-index-with-hash-column-ordering)
```

Comment:
I think that HASH is the default for the first column and HASH for the others

Reply:
Yes indeed! HASH for the first column and ASC for the second. I saw it here : https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_index/#expression
Thank you. I'll change accordingly.